synth.py

```diff
@@ -57,4 +59,12 @@
 # Format quoted strings as plain text.
 s.replace("google/cloud/bigquery_v2/proto/*.py", "[“”]", '``')
 
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+templated_files = common.py_library(cov_level=100)
+# we do not want to override the custom noxfile with the generated one
+os.remove(os.path.join(templated_files, "noxfile.py"))
+s.move(templated_files)
```

Comment:
You can add an excludes list to `s.move`.

```python
s.move(templated_files, excludes=["noxfile.py"])
```

Reply:
Thanks!

Since it's a minor thing, I'll do it separately to not block the main PR of the repo migration.